config/ftbquests/quests/chapters/tools.snbt

```diff
@@ -3158,72 +3216,125 @@
 			y: -17.0d
 			subtitle: "No, wait-it's actually too much signal."
 			description: [
-				"Third tier jetpacks are made of Electrum, "
-				"Gold, Invar and Steel. All shocking materials (but not exactly). Again Tier 2 is required to make Tier 3 one."
+				"Third tier is a bit tricky, as it requires some sort of alloy metal! It also has the biggest variety of jetpacks to chosoe from."
 				""
-				"Gold and Invar are again the same, Steel is a bit better in everything, and Electrum is the best one from tier 3!"
+				"Alloys can be created with use of many things. Thermal Induction Smelter, Tinker's Construct Smeltery or Create Heated Mixing! Check JEI and other quest chapters for specifics."

```

Comment:
`Alloys can be created with use of many things.` -> `Alloys can be created in many ways`